app/models.py

```diff
@@ -2245,14 +2245,21 @@ class BroadcastMessage(db.Model):
     cancelled_at = db.Column(db.DateTime, nullable=True)
     updated_at = db.Column(db.DateTime, nullable=True, onupdate=datetime.datetime.utcnow)
 
-    created_by_id = db.Column(UUID(as_uuid=True), db.ForeignKey('users.id'), nullable=False)
+    created_by_id = db.Column(UUID(as_uuid=True), db.ForeignKey('users.id'), nullable=True)
```

Comment:
i cant leave a comment on the line directly but on line 2294 we should now check if created_by_id is none before calling `str` on it

Reply:
I noticed this for the other optional joins to the `users` table too, have fixed in 78e8785